src/admin/shared/helpers/content-picker/internal-link.ts

```diff
@@ -1,20 +1,32 @@
+import { remove } from 'lodash-es';
+
 import { APP_PATH } from '../../../../constants';
 
 import { PickerSelectItemGroup } from '../../types';
 import { parsePickerItem } from './parse-picker';
 
-const APP_PATH_ARRAY = Object.entries(APP_PATH);
+const APP_PATH_ARRAY = Object.values(APP_PATH);
 
 // Return InternalLinkItems items from APP_PATH
 export const fetchInternalLinks = async (limit: number): Promise<PickerSelectItemGroup> =>
 	parseInternalLinks(APP_PATH_ARRAY, limit);
 
-export const parseInternalLinks = (raw: any, limit: number) => {
-	const filteredItems = raw.slice(0, limit).filter((item: any) => !item[1].includes(':'));
+export const parseInternalLinks = (allPaths: string[], limit: number) => {
+	const paths = allPaths.slice(0, limit).filter((path: any) => !path.includes(':'));
+	remove(paths, '/mijn-werkruimte/opdrachten/maak');
+	paths.push('/mijn-werkruimte');
```

Comment:
Ik zou een array constante aanmaken met de `PATHS_TO_REMOVE` en `PATHS_TO_ADD`. Dan een `forEach` of iets gelijkaardigs om hierover te lopen. Lijkt me properder and meer future-proof.